src/js/media-container.js

```diff
@@ -486,6 +486,7 @@ class MediaContainer extends globalThis.HTMLElement {
       // Setting autohide to -1 turns off autohide
       if (this.autohide < 0) return;
 
+      /** @type {ReturnType<typeof setTimeout>} */
```

Comment:
I think we'd prefer to keep the code less noisy if possible.

isn't it the same signature? on my end TS is not erroring / warning.
yarn build seems to pass as well

Reply:
In my project I have a setup where my templates are type aware, so the types for media-chrome end up being loaded, but I do not have `@types/node` installed so typescript can't find the `NodeJS` namespace and gives me a lint error.

<img width="378" alt="image" src="https://github.com/muxinc/media-chrome/assets/83799/4dfebdc1-59b1-4565-8050-8a45d660639f">

<img width="1055" alt="image" src="https://github.com/muxinc/media-chrome/assets/83799/e48644a1-f225-48a9-8e82-435df12f2c8d">

The signatures are different, in node you get the instance of the Timeout class, in the browser it's just a number.

Reply:
The test runner is bringing in @types/node so you won't see this error when running `tsc`. This will also be the reason it gets typed as `NodeJS.Timeout` to begin with.

<img width="936" alt="image" src="https://github.com/muxinc/media-chrome/assets/83799/6aa4df05-6da6-4f6b-be7c-9c5aa69eabd3">

Reply:
Another option instead of a type comment would be to use `window.setTimeout` as that would indicate to typescript it should use the DOM typings, not Node. But I haven't tested that, and it would probably need a comment anyway to communicate why only this instance of `setTimeout` is using `window` so it's not removed by someone in the future.